R/RMauthClient.R

```diff
@@ -95,7 +95,7 @@ makeMAuthCall<-function(RMauthClientObject, method, base_url, route, queryString
   {
     RETRY(verb = "GET", url = requestURL, config = add_headers(.headers = mAuthHeader), times = retryAttempts)
   } else if (method=="POST"){
-    RETRY(verb = "POST", url = requestURL, config = add_headers(.headers = mAuthHeader), body=body, times = retryAttempts)
+    RETRY(verb = "POST", url = requestURL, config = add_headers(.headers = mAuthHeader), body=body, pause_cap = 60, times = retryAttempts)
```

Comment:
I think you need to declare `pause_cap` as an argument of `makeMAuthCall` function (line 76) and pass it down to `RETRY` function like so:
```suggestion
    RETRY(verb = "POST", url = requestURL, config = add_headers(.headers = mAuthHeader), body=body, pause_cap = pause_cap, times = retryAttempts)
```

Reply:
I have  made the required changes Leandro